test/tests/neutronics/adaptivity/cell.i

```diff
@@ -1,13 +1,14 @@
 [Mesh]
   [sphere]
-    # Mesh of a single pebble with outer radius of 1.5 (cm)
     type = FileMeshGenerator
-    file = ../../neutronics/meshes/sphere.e
+    file = ../meshes/sphere.e
   []
   [solid]
     type = CombinerGenerator
     inputs = sphere
-    positions = '0 0 0'
+    positions = '0 0 0
```

Comment:
Can we make these tests a bit smaller? Like 1 pebble? Some of the output files are 0.5 M.

Reply:
I can shrink the adaptivity tests down to one pebble and drop the block restriction tests to two pebble. That should help with the PR size.